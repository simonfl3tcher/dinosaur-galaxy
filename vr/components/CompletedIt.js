// @flow

// React
import React, { Component } from 'react';
import { Text, View } from 'react-vr';

// Libs
import styles from '../styles/main';

export default class CompletedIt extends Component {
  render() {
    return (
      <View style={styles.gameStyle}>
        <Text style={styles.text}>You have completed all of the questions</Text>
      </View>
    );
  }
}
