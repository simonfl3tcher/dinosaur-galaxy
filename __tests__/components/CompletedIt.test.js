import 'react-native';
import 'react-vr';
import React from 'react';
import CompletedIt from '../../vr/components/CompletedIt.js';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<CompletedIt />).toJSON();
  expect(tree).toMatchSnapshot();
});
