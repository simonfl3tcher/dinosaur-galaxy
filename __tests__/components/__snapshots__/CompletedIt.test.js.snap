// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "transform": Array [
        Object {
          "translate": Array [
            -2.25,
            0,
            0,
          ],
        },
      ],
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    isOnLayer={false}
    style={
      Object {
        "color": "#fff",
        "fontSize": 0.2,
        "textAlign": "center",
        "transform": Array [
          Object {
            "translate": Array [
              0,
              2,
              -5,
            ],
          },
        ],
      }
    }
  >
    You have completed all of the questions
  </Text>
</View>
`;
